plugins/CustomTCMaker.cpp

```diff
@@ -125,13 +134,20 @@ CustomTCMaker::do_configure(const nlohmann::json& /*obj*/)
   //// This parameter controls how many new timestamps are calculated when needed
   //// Currently precalculates events for the next 60 seconds
   //m_sorting_size_limit = 60 * m_conf0>clock_frequency_hz;
+  
+  m_latency_monitoring.store( m_conf->get_latency_monitoring_conf()->get_enable_latency_monitoring() );
```

Comment:
`do_configure` is deprecated in v5.
All the config should be inside of `void init(std::shared_ptr<appfwk::ModuleConfiguration> mcfg);`

Technically there's also a new `conf`, but only in non-`DAQModule` classes I think.